algebra/big_operators.lean

```diff
@@ -277,6 +294,30 @@ begin
     simp [mul_sum] }
 end
 
+open nat
+/-- The binomial theorem -/
+theorem add_pow (x y : β) : ∀ n : ℕ,
+    (x + y) ^ n = (range (succ n)).sum (λ m, x ^ m * y ^ (n - m) * choose n m)
+| 0        := by simp
+| (succ n) :=
+have h₁ : x * (x ^ n * y ^ (n - n) * choose n n) = x ^ succ n * y ^ (succ n - succ n)
+    * choose (succ n) (succ n),
```

Comment:
Break this line around the `=` sign rather than `*` sign.

Reply:
done